src/main/java/com/Elessar/app/server/RegisterHandler.java

```diff
@@ -0,0 +1,56 @@
+package com.Elessar.app.server;
+
+import com.sun.net.httpserver.HttpExchange;
```

Comment:
there seems to be a few unused imports.
can you google how to automatically remove unused imports from your IDE?